app/controllers/giving_profiles_controller.rb

```diff
@@ -0,0 +1,4 @@
+class GivingProfilesController < ApplicationController
```

Comment:
What's the reason for adding a new controller? There's a home controller, why not just add another action to that instead?

Reply:
@mintii might be able to offer a little more information here as well

Reply:
Wanted to split the controllers based on users logged in and not logged in.

Reply:
Splitting up logic is a valid concern. But having a descriptive name is good too. @uuunk can we have a discussion about Controller and routes naming convention/expectations at the meeting? I really think it'll be very helpful for us to get on the same page about this. Over the weekend I put up a question about this [on trello](https://trello.com/c/3Kx7QURQ/48-can-we-talk-about-specific-controller-names-and-route-names-so-were-all-on-the-same-page). Hopefully we can touch on it during the meeting.